docs/DeveloperGuide.md

```diff
@@ -451,52 +463,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
 
 
-**Use case: Deleting a single resident**
+**Use case: UC 7. Deleting a single resident**
 
 **MSS**
-1. Resident moves out of RC4.
-2. User uses the delete command to remove the resident from RC4HDB.
+1. Resident moves out of [**RC4**](#glossary).
+2. User deletes the resident.
```

Comment:
Can we add "from RC4HDB" to the end of the second bullet? I think we should be clear on which steps are interactions with the app 😄